### PR TITLE
Release 8.0.0

### PR DESCRIPTION
**Pull request by shoekstra**
Also includes changelog for 7.2.7.

### CODE REVIEW

**Comment by shoekstra** [comment]
Closes #526.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,50 @@ nagios Cookbook CHANGELOG
 =========================
 This file is used to list changes made in each version of the nagios cookbook.
 
-Unreleased
-----------
-### Improvement
+8.0.0
```
Comment (tas50) [inline CHANGELOG.md]: Toss a date on this so people can see when it came out

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,50 @@ nagios Cookbook CHANGELOG
...
+-----
+### Breaking Changes
 - Drop support/testing for CentOS 6, Fedora OS, FreeBSD
-- Drop test suite for allowed_ips - no functional tests
-- Clean up kitchen config
-- Test Chef 12 and Chef 13 but allow Chef 13 to fail until it's properly supported
+- #522 Drop support for RHEL 5
+- #531 Drop support for Fedora OSes
+- #534 Drop support for CentOS 6, Fedora OS, FreeBSD
+
```
Comment (tas50) [inline CHANGELOG.md]: Mention that we require Chef 12.9 now. I *believe* that a new thing

**Comment by shoekstra** [comment]
Closes #545

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,50 @@ nagios Cookbook CHANGELOG
 =========================
 This file is used to list changes made in each version of the nagios cookbook.
 
-Unreleased
-----------
+8.0.0 (19-09-2017)
```
Comment (tas50) [inline CHANGELOG.md:5]: Dates are hard

**Review by tas50 — APPROVED** [review]

**Pull request merged**